app/views/stats/_barchart.html.erb

```diff
@@ -0,0 +1,77 @@
+
+<script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
+<script>
+var questions = [<%= @questions.present? ? @questions : 0 %>];
+var answers = [<%= @answers %>];
```

Comment:
Yes `@questions` does get all of them. But here in barchart I am calculating the unanswered. It's easy to change back!

Reply:
Great, this is awesome, thanks so much so working on this :tada: